types/src/tests/batch_serde.rs

```diff
@@ -83,17 +83,3 @@ fn test_bincode_serde_batch_message() {
         hex::encode(txes_bytes)
     );
 }
-
-proptest::proptest! {
-
-    #[test]
-    fn test_batch_and_serialized(
-        batch in Batch::arbitrary()
-    ) {
-        let digest = batch.digest();
-        let message = WorkerMessage::Batch(batch);
-        let serialized = bincode::serialize(&message).expect("Failed to serialize our own batch");
-        let digest_from_serialized = serialized_batch_digest(&serialized).expect("Failed to hash serialized batch");
-        assert_eq!(digest, digest_from_serialized);
-    }
```

Comment:
The goal of this is to check the computation of the digest on the serialized form of the batch is equivalent to that on the batch itself. Moreover, we have some benches proving they take the same time. TL;DR, it's important to not lose this test.

Reply:
While taking a look at this i discovered that this file is not included in the module hierarchy and thus is not compiled and these tests are not run on main today.

Reply:
🤦